src/Orchard.Web/Core/Title/Settings/TitlePartSettings.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel;
+using System.Linq;
+using System.Web;
+
+namespace Orchard.Core.Title.Settings {
+    public class TitlePartSettings {
+        [Range(0, 1024)]
```

Comment:
why limit the limit?

Reply:
Maximum length for the Title property of the underlying record is set to 1024 characters both in the migration and the record itself already.

Reply:
Whoever wrote this should have use a constant and shared it. Hint:

![image](https://github.com/OrchardCMS/Orchard/assets/1165805/8396a2cd-9ce5-493b-b2a2-9357d5ae1edf)

Reply:
I changed that value to a constant and modified the corresponding recipe too.